kong/plugins/key-auth/handler.lua

```diff
@@ -17,13 +17,20 @@ local KeyAuthHandler = BasePlugin:extend()
 
 KeyAuthHandler.PRIORITY = 1000
 
+local OPTIONS = "OPTIONS"
```

Comment:
I would love for many plugins included in Kong to not introduce such weird practices that are then copy/pasted by plugins authors all over the place. That's ok tho, to be removed ideally.